src/state/sync/RemotePromoSheetSync.tsx

```diff
@@ -6,7 +6,7 @@ import { GetPromoSheetCollectionQuery, PromoSheetOrder } from '@/graphql/__gener
 import { useRunChecks } from '@/components/remote-promo-sheet/runChecks';
 import { IS_TEST } from '@/env';
 
-const RemotePromoSheetSyncComponent = () => {
+const RemotePromoSheetSyncComponent = ({ walletReady }: { walletReady: boolean }) => {
```

Comment:
you could select the `walletReady` directly from redux state within this component instead of passing in as a prop, and then you wouldn't need to memo

Reply:
oh yeah good idea